applications/CompressiblePotentialFlowApplication/python_scripts/define_wake_process_2d.py

```diff
@@ -164,7 +167,7 @@ def ComputeDistancesToWake(self, elem):
         return nodal_distances_to_wake
 
     @staticmethod
```

Comment:
Which is the purpose of this decorator? I've never used it.

Reply:
To define the method as static, meaning that it cannot access the `self` operator, so it can't read/modify any member variable or method from the class.

Reply:
Good to know! :+1: